datalab/mlalpha/_job.py

```diff
@@ -68,6 +51,61 @@ def describe(self):
     job_yaml = yaml.safe_dump(self._info, default_flow_style=False)
     print job_yaml
 
+  @staticmethod 
+  def submit_training(job_request, job_id=None):
+    """Submit a training job.
```

Comment:
Done on comments. I think we only expand args if it is a dict. If it is a list we don't mess up with it.